src/fparser/one/tests/bad_char.f90

```diff
@@ -0,0 +1,55 @@
+MODULE dynadv_cen2
```

Comment:
Please add license/copyright. Since this is straight out of NEMO we should be including their license text. We could also chop this example down.

Reply:
I've cut the example down so it is no longer nemo code (apart from one character!) and added our licence/copyright info.